scripts/ci/Dockerfile

```diff
@@ -21,7 +21,7 @@ ARG APP
 ENV APP=${APP}
 ENV NODE_ENV=production
 
-RUN yarn run build ${APP} --prod
+RUN yarn nx run ${APP}:build:production
```

Comment:
I think this is identical change

Reply:
They are, this shouldve been removed. Ill fix it.